src/Adapter/Carrier/CommandHandler/EditCarrierHandler.php

```diff
@@ -79,8 +79,8 @@ public function handle(EditCarrierCommand $command): CarrierId
         }
 
         // Shipping information
-        if (null !== $command->isShippingHandling()) {
-            $carrier->shipping_handling = $command->isShippingHandling();
+        if (null !== $command->hasAdditionalHandlingFee()) {
```

Comment:
Isn't this a bit redundant with the following lines L98-L107, couldn't it be united in the same block for more clarity?